package.json

```diff
@@ -20,23 +20,26 @@
   "devDependencies": {
     "babel-preset-es2015": "^6.6.0",
     "babelify": "^7.2.0",
-    "chai": "^3.5.0",
-    "chai-as-promised": "^6.0.0",
-    "dependency-check": "^2.8.0",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "dependency-check": "^3.3.0",
     "eslint": "^3.16.1",
     "eslint-config-wunderflats": "^0.4.0",
-    "husky": "^0.13.2",
-    "mocha": "^3.2.0",
+    "eslint-plugin-mocha": "^5.3.0",
+    "eslint-plugin-promise": "^4.1.1",
+    "eslint-plugin-standard": "^4.0.0",
+    "husky": "^2.3.0",
+    "mocha": "^6.1.4",
     "plop": "^1.5.0",
-    "sinon": "^1.17.3"
+    "sinon": "^7.3.2"
   },
   "dependencies": {
     "async-q": "^0.3.1",
-    "camelcase": "^4.0.0",
+    "bson": "^4.0.2",
```

Comment:
Use [bson-objectid](https://www.npmjs.com/package/bson-objectid) instead because `bson` package is really big: https://bundlephobia.com/result?p=bson@4.0.2

Remember that this module is meant to be used browser-side so bundle size matters.

Reply:
Good point. I changed the module.